tests/test_structure/test_all_structures.py

```diff
@@ -0,0 +1,18 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.structure.structures_fixtures import (
+    structures_query,
+    expected_structures_query_response
+)
+
+
+class TestAllStructures(BaseTestCase):
+
+    def test_all_structures(self):
```

Comment:
Rename this to `test_query_all_structures_works`, same to other tests. Function names should be descriptive enough to encourage readability.

Reply:
@Bonifase changes have been made